src/App.js

```diff
@@ -1,23 +1,15 @@
-import logo from './logo.svg';
 import './App.css';
+// import PropTypes from 'prop-types';
+// import Big from 'big.js';
```

Comment:
- [ ] [OPTIONAL]  Inline comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history.